packages/discv5/src/util/toBuffer.ts

```diff
@@ -1,9 +1,4 @@
-export function toBuffer(arr: Uint8Array): Buffer {
-  if (arr instanceof Buffer) return arr;
-  return Buffer.from(arr.buffer, arr.byteOffset, arr.length);
-}
-
-export function numberToBuffer(value: number, length: number): Buffer {
+export function numberToBuffer(value: number, length: number): Uint8Array {
```

Comment:
Is this function still used? maybe delete it or refactor the usage of Buffer here

Reply:
Good catch.  Not sure how I missed that but will do.